ci-operator/step-registry/optional-operators/operator-sdk/optional-operators-operator-sdk-ref.yaml

```diff
@@ -15,10 +15,10 @@ ref:
       comma separated list of namespaces (for some mode type).
       values can be:
       
-      * OwnNamespace - indicates that the operator can be a member of an `OperatorGroup` that selects its own namespace. Must not contain list of namespaces.
-      * SingleNamespace=<namespace name> - indicates that the operator can be a member of an `OperatorGroup` that selects one namespace. Must contain one and only one namespace. the namespace must be different than the OO_INSTALL_NAMESPACE
-      * MultiNamespace=<namespace name 1>[,<namespace name 2>[,...]] - indicates that the operator can be a member of an `OperatorGroup` that selects more than one namespace. The namespace list must not be empty.
-      * AllNamespaces - (default) indicates that the operator can be a member of an `OperatorGroup` that selects all namespaces (target namespace set is the empty string ""). The namespace list must be empty.
+      * `OwnNamespace` - indicates that the operator can be a member of an `OperatorGroup` that selects its own namespace. Must not contain list of namespaces.
+      * `SingleNamespace=<namespace name>` - indicates that the operator can be a member of an `OperatorGroup` that selects one namespace. Must contain one and only one namespace. the namespace must be different than the OO_INSTALL_NAMESPACE
```

Comment:
if you're adding back quotes to all terms, maybe add it to `OO_INSTALL_NAMESPACE`as well?

Reply:
and add a period at the end